api/src/cli/index.test.ts

```diff
@@ -1,25 +1,30 @@
 import { Command } from 'commander';
 import { Extension, HookConfig } from '@directus/shared/types';
 import { createCli } from './index';
+import { test, expect, vi, beforeEach } from 'vitest';
```

Comment:
```suggestion
import { test, expect, vi, beforeEach, describe } from 'vitest';
```

Pro-tip: Temporarily remove the package `@type/jest` in the root, so the namespace in the IDE is not polluted and you will get a warning when an import from `vitest` is missing